docs/content/entity-templates.md

```diff
@@ -0,0 +1,62 @@
+| `SetPersistence(bool persistence)`                           | Specify whether your entity should be saved in snapshots. For more information, see the documentation on [persistence](https://docs.improbable.io/reference/latest/shared/glossary#persistence). |
+| `SetReadAcl(string attribute, param string[] attribute)` or `SetReadAcl(List<string> attribute)` | Specify which worker type(s) can have [read access](https://docs.improbable.io/reference/latest/shared/glossary#read-and-write-access-authority) to the entity. |
+| `SetEntityAclComponentWriteAccess(string attribute)`         | Specify which worker type can have authority over the [`EntityAcl`](https://docs.improbable.io/reference/latest/shared/schema/standard-schema-library#entityacl-required) component of your entity. This is useful if, while the game is running, you want to change which worker type(s) can have authority over the entity's other components. |
+| `AddComponent(ComponentData data, string writeAccess)`       | Add a user-defined component to your entity and specify which worker type(s) can have authority over it. |
+| `Build()`                                                    | Create a finished entity template.                           |
+
+For each [schema component](https://docs.improbable.io/reference/13.2/shared/glossary#schema) you define, [the code generator](./code-generator.md) generates a helper method to create a `ComponentData` object for that component. For example, for the following schema:
+
+```
+component Health {
+  id = 1;
+  int32 health_value = 1;
+}
+```
+
+There is a method `Health.Component.CreateSchemaComponentData(int healthValue)` which will instantiate and populate a `ComponentData` object.
+
+Below is an example of how to define an entity template using the `EntityBuilder`. In this example we will be creating a template for a creature:
+
+```csharp
+public static class CreatureTemplate
+{
+    public static Entity CreateCreatureEntityTemplate(Coordinates coords)
+    {
+        const string entityType = "Creature";
+
+        var healthComponent = Health.Component.CreateSchemaComponentData(healthValue: 100);
+
+        var entity = EntityBuilder.Begin()
+            .AddPosition(coords.X, coords.Y, coords.Z, WorkerUtils.UnityGameLogic)
+            .AddMetadata(entityType, WorkerUtils.UnityGameLogic)
+            .SetPersistence(true)
+            .SetReadAcl(WorkerUtils.AllWorkerAttributes)
+            .AddComponent(healthComponent, WorkerUtils.UnityGameLogic)
+            .Build();
+
+        return entity;
+    }
+}
+```
+
+We have created an entity template, now we should spawn this entity. This can be done either using the [MonoBehaviour](gameobject/world-commands.md) or [ECS](ecs/world-commands.md) workflow.
```

Comment:
Reword first sentence to `You can use this entity template to spawn a Creature entity.` or similar